portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +25,30 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
+  private List<String> snailFacts;
```

Comment:
This is fine. If this facts won't be changed after init(). Its better to have 
```suggestion
  private static final List<String> snailFacts = ImmutableList.of(factString1,factString2, ...);
```